boards/arm/stm32_blackpill_v2/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 Brian Bradley
+# SPDX-License-Identifier: Apache-2.0
+
+# Black Pill V2.0 board configuration
+
+config BOARD_BLACKPILL_V2
```

Comment:
When possible, the Kconfig symbol matching the target should be the same as the directory used for the board, so `BOARD_STM32_BLACKPILL_V2`.
I mention this for reference as I prefer the name changed to more specific "blackpill_f411ce", cf global comment.

Reply:
> Thanks for this addition!
> 
> Looking to following site https://stm32-base.org/boards/, there is a number of blue/black/green pills from various vendors and I think we should use a more specific name to avoid confusion on these boards. We'd benefit from a more specific name, such as blackpill_f411ce as used on [stm32duino project](https://github.com/stm32duino/Arduino_Core_STM32).
> 
> Also it would be nice to configure dfu-utils runner to enable flashing using west.

No problem! This is my first full board addition so I may be missing some things. 

But, I thought that I did setup the dfu-util runner, and I tested it to work perfectly with ```west flash```

The board.cmake includes:

```
board_runner_args(dfu-util "--pid=0483:df11" "--alt=0" "--dfuse")

include(${ZEPHYR_BASE}/boards/common/dfu-util.board.cmake)
```
Which I thought was doing as you said.

You are correct on the board name, I probably should have given that more thought.

Reply:
Ok, I oversaw the board.cmake file. This is fine then, but please update doc to reflect that west can be used, something like follows:
.. zephyr-app-commands::
   :zephyr-app: samples/hello_world
   :board: 96b_carbon
   :goals: build flash

Reply:
> When possible, the Kconfig symbol matching the target should be the same as the directory used for the board, so `BOARD_STM32_BLACKPILL_V2`.
> I mention this for reference as I prefer the name changed to more specific "blackpill_f411ce", cf global comment.

Fixed.